packages/mdc-menu/component.ts

```diff
@@ -143,6 +143,14 @@ export class MDCMenu extends MDCComponent<MDCMenuFoundation> {
     this.menuSurface_.setAnchorMargin(margin);
   }
 
+  /**
+   * Sets the list item as the selected row at the specified index.
+   * @param index Index of list item within menu.
+   */
+  setSelectedIndex(index: number) {
```

Comment:
Do you think component should also support `getSelectedIndex` API? Seems like a common use case to retrieve current state.

`getSelectedIndex` will always return array of selected index to support multiple selection group. We can track this as separate issue.

Reply:
https://github.com/material-components/material-components-web/issues/4634